content/docs/cloud/roadmap.md

```diff
@@ -1,9 +1,13 @@
----
-title: Roadmap
+# <<<<<<< HEAD
```

Comment:
conflict needs to be resolved before the merge

Reply:
I'm having a conflict I believe with `content/docs/cloud/roadmap.md` since I moved roadmap to a different folder. 

Any thoughts on how to fix this?

I realize you are still updating `roadmap.md` in its original folder, cloud.